ArduPlane/mode.h

```diff
@@ -363,6 +364,11 @@ class ModeGuided : public Mode
     // handle a guided target request from GCS
     bool handle_guided_request(Location target_loc) override;
 
+#if AP_PLANE_OFFBOARD_GUIDED_SLEW_ENABLED
+    // handle a guided airspeed command, typically from companion computer
+    MAV_RESULT handle_change_airspeed(const float airspeed, const float acceleration);
```

Comment:
```suggestion
    MAV_RESULT handle_change_airspeed(const float airspeed, const float acceleration);
```
MAV_RESULT could be a `bool` since it's either accepted or denied. Depends if we want mavlink error codes here or not.

Reply:
Yes, this should be bool.